infra/feast-operator/internal/controller/featurestore_controller_pvc_test.go

```diff
@@ -69,6 +69,7 @@ var _ = Describe("FeatureStore Controller-Ephemeral services", func() {
 		onlineStoreMountPath := "/online"
 		registryMountPath := "/registry"
 
+		accessModes := []corev1.PersistentVolumeAccessMode{corev1.ReadWriteMany}
```

Comment:
can we test setting multiples here?
```suggestion
		accessModes := []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce, corev1.ReadWriteMany}
```

Reply:
Done and test is passing with two.